src/api/libcellml/variable.h

```diff
@@ -65,6 +65,25 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
     static void addEquivalence(const VariablePtr &variable1, const VariablePtr &variable2);
 
     /**
+     * @brief Remove each argument variable to the other's equivalent variable set.
+     * Removes a copy of @p variable1 from the set of equivalent variables for
+     * @p variable2 if present. Also removes a copy of @p variable2 from the
```

Comment:
Nitpicking: "... variable2**,** if present..." (comma). (Same on the next line.)